tools/image-base

```diff
@@ -19,14 +19,25 @@ docker build --progress plain --tag ${OWNER}/ubuntu-certs --file Dockerfile.cert
 
 cd ~
 if [[ ! -d docker-stacks ]]; then
-   git clone https://github.com/jupyter/docker-stacks.git
-   cd docker-stacks
+    git clone https://github.com/jupyter/docker-stacks.git
+    cd docker-stacks
 else
     cd docker-stacks
     git fetch origin
     git rebase origin/master
 fi
 
+# checkout the appropriate docker-stacks revision if using frozen specs;
```

Comment:
If desired,  you can streamline this some using ${IMAGE_DIR} to replace: ${JUPYTERHUB_DIR}/deployments/$DEPLOYMENT_NAME/image

Reply:
Done.